datasource/observer/clients/configurableobserverclient.js

```diff
@@ -0,0 +1,101 @@
+/*
+ * Copyright (c) 2016-present, IBM Research
+ * Licensed under The MIT License [see LICENSE for details]
+ */
+
+'use strict';
+
+const ObserverClient = require ('./observerclient');
+const request        = require("request");
+const Promisify      = require("ninja-util/promisify");
+
+class ConfigurableObserverClient extends ObserverClient
```

Comment:
Why keep both classes?

Reply:
I believe it makes more sense to isolate the new behavior in a specialized class.

Reply:
Seems fair.